tables/zh_TW.tbl

```diff
@@ -1,16 +1,17 @@
-#-index-name: Mandarin, Taiwan
-#-display-name: Taiwanese braille
+#-index-name: Mandarin, Taiwan, bopomofo, zhuyin
+#-display-name: Taiwan bopomofo braille
 
 #+language:cmn
 #+region:cmn-TW
 #+type:literary
+#+system:cmn-bopomofo
 # Marked as "direction:forward" by Bue Vester-Andersen
 # as there are only very few backward tests.
 #+direction:forward
```

Comment:
@school510587 This comment was added by Bue. But since you are the author of the table, can you confirm it is indeed forward only, or also backward? The comment by Bue can be removed.